robots/FrankaEmikaPandaRobot.m

```diff
@@ -0,0 +1,55 @@
+% panda = FrankaEmikaPandaRobot.kinematics() returns a DQ_kinematics object
+% using the modified Denavit-Hartenberg parameters of the Franka Emika
+% Panda robot
+
+% (C) Copyright 2015-2022 DQ Robotics Developers
+% 
+% This file is part of DQ Robotics.
+% 
+%     DQ Robotics is free software: you can redistribute it and/or modify
+%     it under the terms of the GNU Lesser General Public License as published by
+%     the Free Software Foundation, either version 3 of the License, or
+%     (at your option) any later version.
+% 
+%     DQ Robotics is distributed in the hope that it will be useful,
+%     but WITHOUT ANY WARRANTY; without even the implied warranty of
+%     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%     GNU Lesser General Public License for more details.
+% 
+%     You should have received a copy of the GNU Lesser General Public License
+%     along with DQ Robotics.  If not, see <http://www.gnu.org/licenses/>.
+%
+% DQ Robotics website: dqrobotics.github.io
+%
+% Contributors to this file:
+%     Frederico Fernandes Afonso Silva - frederico.silva@ieee.org
+
+classdef FrankaEmikaPandaRobot
+    methods (Static)
+        function ret = kinematics()
+            % Modified D-H of Franka Emika Panda
+            FEp_DH_theta = [0, 0, 0, 0, 0, 0, 0];
+            FEp_DH_d = [0.333, 0, 3.16e-1, 0, 3.84e-1, 0, 0];
+            FEp_DH_a = [0, 0, 0, 8.25e-2, -8.25e-2, 0, 8.8e-2];
+	        FEp_DH_alpha = [0, -pi/2, pi/2, pi/2, -pi/2, pi/2, pi/2];
+            FEp_DH_type = repmat(DQ_SerialManipulatorDH.JOINT_ROTATIONAL,1,7);
+
+	        FEp_DH_matrix = [FEp_DH_theta;
```

Comment:
Please adjust the indentation.

Reply:
Fixed in commit [21a2174](https://github.com/ffasilva/matlab/commit/21a2174bb008cb2fa9cf690f6a1206d1f54126a6).